lib/widgets/post_list_options.dart

```diff
@@ -1,78 +1,41 @@
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_hooks/flutter_hooks.dart';
 import 'package:lemmy_api_client/v2.dart';
 
-import 'bottom_modal.dart';
+import 'radio_picker.dart';
 
 /// Dropdown filters where you can change sorting or viewing type
-class PostListOptions extends HookWidget {
-  final void Function(SortType sort) onChange;
-  final SortType defaultSort;
+class PostListOptions extends StatelessWidget {
+  final ValueChanged<SortType> onSortChanged;
+  final SortType sortValue;
```

Comment:
It is `sortValue` and `onSortChanged` instead of `value` and `onChanged` because in the future this widget will also handle style changes (`styleValue`, `onStyleChanged`)